README.md

```diff
@@ -1,2 +1,11 @@
-# demo-app
-Repository for the app used in the Rode Demo 
+# Demo Application and CI Pipeline for Rode
+This is the application repository used to demo Rode capabilities.
+- The application used in the demo is a simple Express/Node app serving a `Hello World` response. [server.js](server.js)
```

Comment:
Feels a little awkward to end the line with the filename, maybe have "application" be the link text or something along those lines.

Reply:
Yeah - it's kind of just hanging out there. I'll change that.